### ISSUE TITLE
to wide x and 

### ISSUE THREAD

**Issue by srem1** [Meshinger/Meshinger#1]
Meybe add solar panel in front , a little short x dimension and put second socket for AA,AAA, 18650 etc, step up down from 1 to 24V
for example in emergency situation I have different source of power, meybe battery (europe acumulator) meybe AA, beybe different size /capacity etc. I would like to use separately power not both

**Comment by awgh** [comment]
Yeah this has been the most common feedback.  I'm redoing the board design again and removing the battery entirely.  I'll put on a more robust power connector and then people can use whatever kind of power source they want.  I'm also going to revisit the buck regulator and swap it out for a boost-buck to make the USB port work and some other changes.

I'll take the 1-24V range into consideration, but I might have to back down to 1-18V or similar depending on which boost-buck I go with.  I'll leave this issue open until the change is checked in.
